lua/package-info/config.lua

```diff
@@ -21,9 +17,25 @@ local DEFAULT_OPTIONS = {
     },
     autostart = true,
 }
+if vim.o.termguicolors then
```

Comment:
I think it would be cleaner if the hex colors remained the default, and the under there would just be 
```lua
if not vim.o.termguicolors then 
-- set colors and param
end
```

Also, I think the code is pretty self-explanatory so no need for comments.